src/shared/components/query/QueryStoreUtils.ts

```diff
@@ -65,8 +66,8 @@ export function nonMolecularProfileParams(store:QueryStore):NonMolecularProfileQ
         Action: 'Submit',
     };
 
-    if (store.selectedStudyIds.length !== 1) {
-        ret = Object.assign(ret, { cancer_study_list: store.selectedStudyIds.join(",") });
+    if (selectedStudyIds.length !== 1) {
+        ret = Object.assign(ret, { cancer_study_list: selectedStudyIds.join(",") });
```

Comment:
why are we using assign instead of just ret.cancer_study_list = ...   ?

Reply:
I think that may be legacy from when I was assigning more than one member here.